app/models/feed.rb

```diff
@@ -19,6 +19,11 @@ class Feed < BaseModel
   has_many :feed_tokens, autosave: true, dependent: :destroy
   alias_attribute :tokens, :feed_tokens
 
+  has_one :feed_image, -> { complete.order('created_at DESC') }, autosave: true, dependent: :destroy
```

Comment:
Fixes #426.  This `feed_image` will return the _most recent completed_ image.  So any new images will end up in the has_many, but not here until processing completes.